packages/editor-ui/src/stores/settings.store.ts

```diff
@@ -32,6 +31,7 @@ export const useSettingsStore = defineStore(STORES.SETTINGS, () => {
 	});
 	const templatesEndpointHealthy = ref(false);
 	const api = ref({
+		apiKeysLimit: 0,
```

Comment:
Or 1 to have the same limit as in the BE?

Reply:
I guess it does not matter and this will always get the value we get from the FE.